Gems/AWSMetrics/cdk/aws_metrics/aws_utils/resource_name_sanitizer.py

```diff
@@ -0,0 +1,42 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import hashlib
+
+MAX_RESOURCE_NAME_LENGTH_MAPPING = {

```

Comment:
Ideally we should link to the AWS docs or how these are set.

Reply:
Update the function description based on the discussion offline.